extensions/assessment/package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "assessment",
```

Comment:
This extension needs to be added to https://github.com/microsoft/azuredatastudio/blob/master/build/lib/extensions.ts#L233 so it's not bundled as a built-in extension

Reply:
@Charles-Gagnon since sql-assessment is in the external-extensions list - it ships separately from ADS. I suppose we should somehow be available in the marketplace. Could you please advise the steps to get to the marketplace?